modules/openapi-generator/src/test/resources/3_0/petstore.yaml

```diff
@@ -116,6 +116,13 @@ paths:
             type: array
             items:
               type: string
+        - name: maxCount
+          in: query
+          description: Maximum number of items to return
+          required: false
+          schema:
+            type: integer
+            format: int32
```

Comment:
We'll remove this as we want to keep 

Additional test cases should go to https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/test/resources/3_0/petstore-with-fake-endpoints-models-for-testing.yaml instead